packages/amplify-graphql-transformer-interfaces/src/graphql-api-provider.ts

```diff
@@ -128,6 +128,7 @@ export type MappingTemplateProvider = InlineMappingTemplateProvider | S3MappingT
 
 export interface GraphQLAPIProvider extends IConstruct {
   readonly apiId: string;
+  readonly graphqlUrl: string;
```

Comment:
I don't actually see this accessed or mutated anywhere in this PR. Am I missing its usage, or will it be consumed in a followup?

Reply:
This change shouldn't have been included here. Removed it in [65451d6](https://github.com/aws-amplify/amplify-category-api/pull/2812/commits/65451d6fb963abc4c3232cbcc77b65b4d697015c) and will include it in a subsequent relevant PR. Thanks for calling that out.